app/services/reporting/total_user_count_report.rb

```diff
@@ -10,14 +10,15 @@ def initialize(report_date = Time.zone.today)
 
     def total_user_count_report
       [
-        ['All-time user count'],
-        [total_user_count],
+        ['All-time user count', total_user_count],
+        ['Total verified users', verified_user_count],
+        ['Total annual users', annual_total_user_count],
```

Comment:
one more idea, maybe we add columsn (or rows) that say when "annual" starts & ends so if we combine data points from seprate reports its clear?